cmd/kube-proxy/app/server.go

```diff
@@ -169,6 +169,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 		"NAT timeout for TCP connections in the CLOSE_WAIT state")
 	fs.BoolVar(&o.config.EnableProfiling, "profiling", o.config.EnableProfiling, "If true enables profiling via web interface on /debug/pprof handler.")
 	fs.StringVar(&o.config.IPVS.Scheduler, "ipvs-scheduler", o.config.IPVS.Scheduler, "The ipvs scheduler type when proxy mode is ipvs")
+	fs.StringSliceVar(&o.config.NodePortAddresses, "nodeport-addresses", o.config.NodePortAddresses,
+		"Values are as a parameter to select the interfaces where nodeport works. Values must be either IP blocks or `default-route` or ``(empty), default to empty covering all network interfaces")
```

Comment:
Is this a comma-separated list?  Say so.

"A comma-separated list of values which specify the addresses to use for NodePorts.  Values may be either IP blocks (e.g. 10.128.0.0/16), or \"default-route\" (meaning whichever interface is the default route).  The default (\"\") means to use all local addresses.:

Reply:
> Is this a comma-separated list?

Not really... It's a string slice, but I have fixed the comment message correspondingly, THANKS. 

"A string slice of values which specify the addresses to use for NodePorts. Values may be valid IP blocks (e.g. 1.2.3.0/24) or IP addresses(e.g. 1.2.3.4). The default empty string slice ([]) means to use all local addresses."